docs/components/modeler/bpmn/timer-events/timer-events.md

```diff
@@ -83,7 +83,7 @@ It's possible to define a start time. By doing this, the timer triggers for the
 - `R5/PT10S`: Every 10 seconds, up to five times
 - `R/P1D`: Every day, infinitely
 - `R/2022-01-01T10:00:00+02:00[Europe/Berlin]/P1D`: Every day infinitely, starting from January 1, 2022 at 10 a.m. UTC plus 2 hours
-- `R3/2022-04-027T17:20:00+02:00Z/P1D`: Every day up to three times, starting from April 27, 2022 at 5:20 p.m. UTC
+- `R3/2022-04-027T17:20:00Z/P1D`: Every day up to three times, starting from April 27, 2022 at 5:20 p.m. UTC
```

Comment:
❌ There are mistakes in the examples

I was curious about this and did some extra testing.

>`R3/2022-04-027T17:20:00Z/P1D` 

fails because the day of the month is `027`

>R/2022-01-01T10:00:00+02:00[Europe/Berlin]/P1D

fails because it specifies both a time offset as well a zone.

I was able to deploy models to `8.1.0-alpha3` with:
```
R3/2022-07-29T14:25:00Z/PT10S
R3/2022-07-29T14:25:00+02:00/PT10S
R3/2022-07-29T14:25:00Z[Europe/Berlin]/PT10S
```

Strangely, none of the cycles triggered when I expected on SaaS 🤔 

<img width="405" alt="Screen Shot 2022-07-29 at 14 30 35" src="https://user-images.githubusercontent.com/3511026/181758749-00215cdd-bcc6-482a-8cd1-8390e0b97bc5.png">

Reply:
>Strangely, none of the cycles triggered when I expected on SaaS 🤔

✅ Ah I've figured it out, it was a timezone mistake on my side 🤷